### ISSUE TITLE
Unexpected behavior in Crud with aside editor

### ISSUE THREAD

**Issue by tulioag** [vaadin/vaadin-crud-flow#165]
- Editing an item then clicking in `New Item`, the title in the aside remains `New Item`. Clicking a second time in `New item` makes the button disappear. 
- Clicking in `New Item` then clicking in an item in the grid keeps the title as `New Item`.

![Crud_Strange_Behavior](https://user-images.githubusercontent.com/28783969/67936469-84227500-fbd4-11e9-91cb-7171ba465741.gif)

Code:
```java
import java.io.Serializable;
import java.util.List;
import java.util.Random;
import java.util.stream.Collectors;
import java.util.stream.IntStream;

import org.apache.commons.lang3.RandomStringUtils;

import com.vaadin.flow.component.crud.BinderCrudEditor;
import com.vaadin.flow.component.crud.Crud;
import com.vaadin.flow.component.crud.CrudEditorPosition;
import com.vaadin.flow.component.grid.Grid;
import com.vaadin.flow.component.orderedlayout.VerticalLayout;
import com.vaadin.flow.data.binder.Binder;
import com.vaadin.flow.data.provider.ListDataProvider;
import com.vaadin.flow.router.Route;
import com.vaadin.flow.server.PWA;

@Route("")
@PWA(name = "My Application", shortName = "My Application")
public class MainView extends VerticalLayout {

	public MainView() {
		Binder<Bean> binder = new Binder<Bean>();
		BinderCrudEditor<Bean> editor = new BinderCrudEditor<>(binder);
		Grid<Bean> grid = new Grid<>(Bean.class);
		Crud.addEditColumn(grid);
	
		Crud<Bean> crud = new Crud<>(Bean.class,grid, editor);
		crud.setEditorPosition(CrudEditorPosition.ASIDE);
		crud.setEditOnClick(true);
		add(crud);
		crud.setDataProvider(new ListDataProvider<Bean>(getBeans()));
	}

	private static List<Bean> getBeans() {
		Random random = new Random();
		return IntStream.range(0, 1000).mapToObj(
				i -> new Bean(RandomStringUtils.randomAlphabetic(10), RandomStringUtils.randomAlphabetic(10), random.nextInt(100)))
				.collect(Collectors.toList());
	}


    public static class Bean implements Serializable {

        private String firstName;
        private String lastName;
        private int age;
        
        public Bean() {}
        
        public Bean(String firstName, String lastName, int age) {
            super();
            this.firstName = firstName;
            this.lastName = lastName;
            this.age = age;
        }

        public String getFirstName() {
            return firstName;
        }
        public void setFirstName(String firstName) {
            this.firstName = firstName;
        }
        public String getLastName() {
            return lastName;
        }
        public void setLastName(String lastName) {
            this.lastName = lastName;
        }
        public int getAge() {
            return age;
        }
        public void setAge(int age) {
            this.age = age;
        }

    }
}
```

**Comment by alvarezguille** [comment]
Editor title was reported in https://github.com/vaadin/vaadin-crud/issues/183

**Comment by web-padawan** [comment]
The issue linked above was fixed by vaadin/vaadin-crud#184. Closing.
